xcape/cape_fortran.py

```diff
@@ -1,14 +1,26 @@
 
```

Comment:
The code below has a if loop for model level or pressure level.
The two CAPE_CODE_* are almost the same. But CAPE_CODE_pressure_lev has a (n2) array that identifies the position of the vertical level that is above surface pressure (where p <= ps)